packages/atlas-service/src/components/ai-signin-modal.tsx

```diff
@@ -26,13 +27,16 @@ type SignInModalProps = {
 };
 
 const titleStyles = css({
-  marginBottom: spacing[3],
+  marginBottom: spacing[400],
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
 });
 
 // For whatever reason leafygreen marketing modal doesn't have the same spacing
 // between image and title in dark mode
 const titleDarkModeStyles = css({
-  marginTop: spacing[4],
+  marginTop: spacing[600],
```

Comment:
Looks like this was fixed in the LeafyGreen update. We can remove this dark mode override now, currently it looks a bit funny because of too much spacing in dark mode.